src/arch/linux/async/signal.c

```diff
@@ -450,9 +452,8 @@ static void leavedos_emerg(int sig, siginfo_t *si, void *uc)
 {
   ucontext_t *uct = uc;
   sigcontext_t *scp = &uct->uc_mcontext;
-  init_handler(scp, uct->uc_flags);
+  init_handler0(scp);
```

Comment:
Disabled code, under if 0.

Reply:
yes I know, just wanted it to be consistent in case you want to re-enable it in the future.
(I thought the only purpose of #if 0 is that you can easily change to #if 1 for testing, otherwise you can just delete it as git still has it)